### DISCUSSION TITLE
function to merge classes, without variants?

### DISCUSSION THREAD

**Discussion by vipindigiqt** [heroui-inc/tailwind-variants#149] [Q&A]
Currently, I'm using CVA in my project.

It got a custom function `cx()` like this to merge the classNames without defining any variants.

```js
export const { cva, cx, compose } = defineConfig({
    hooks: {
        onComplete: (className) => twMerge(className),
    },
})
```

How to do it with tailwind-variants?

**Comment by mskelton** [comment]
Why not just use `tailwind-merge` directly?  Looks like you already have it installed in your project.
